src/Html/Builder.php

```diff
@@ -538,11 +563,20 @@ public function addIndex(array $attributes = [])
      */
     public function pipeline($url, $pages)
     {
+        $url .= $this->getQueryString();
+
         $this->ajax = "$.fn.dataTable.pipeline({ url: '{$url}', pages: {$pages} })";
 
         return $this;
     }
 
+    public function getQueryString()
```

Comment:
Sorry for late response, I think this part may cause some breaking change? What if we already have a query string on our default URL? 

```php
'ajax' => '/data?user_id=1',
...
```

Reply:
Exactly, I had that issue and also updated accordingly. I am currently so overwhelmed by projects, sadly don't have time to re-think some code and push them.